.github/workflows/deploy.yaml

```diff
@@ -27,7 +27,19 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+      - name: Version
+        # @see https://github.com/lerna/lerna/tree/main/commands/version#readme
+        # Bump version of packages changed since the last release
+        ## `./bin/ci/semver.sh` - determines semantic versioning keyword, e.g.: major, minor, patch
+        ## --yes` - skip all confirmation prompts
+        run: npx lerna version ${./bin/ci/semver.sh} --yes
```

Comment:
Well, you are probably right. I set the command to `yarn lerna` to use internally installed lerna.

Also refactored this command as well and move it to `Makefile` because I think it is a better place where to store and comment release commands.